libpod/container_api.go

```diff
@@ -495,41 +498,111 @@ func (c *Container) WaitWithInterval(waitTimeout time.Duration) (int32, error) {
 	}
 	chWait := make(chan error, 1)
 
-	defer close(chWait)
```

Comment:
It seems like this could still be useful, given that `ctx.Done()` is a goroutine and not guaranteed to fire at the end of the function, but rather when the entire handler returns, AFAIK

Reply:
Probably shouldn't hurt, but also it's not necessary to close a channel here.